tests/fdb/test_fdb_flush.py

```diff
@@ -0,0 +1,362 @@
+            ptfhost,
+            "ptftests",
+            testCase,
+            platform_dir="ptftests",
+            params=testParams,
+            log_file="/tmp/{0}".format(testCase),
+            is_python3=True
+        )
+
+    @pytest.fixture(scope="class", autouse=True)
+    def copyFdbInfo(self, duthosts, rand_one_dut_hostname, ptfhost, tbinfo):
+        """
+            Compies FDB info file to PTF host
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+                ptfhost (AnsibleHost): Packet Test Framework (PTF)
+
+            Returns:
+                None
+        """
+        duthost = duthosts[rand_one_dut_hostname]
+        mgFacts = duthost.get_extended_minigraph_facts(tbinfo)
+        ptfhost.host.options['variable_manager'].extra_vars.update({
+            "minigraph_vlan_interfaces": mgFacts["minigraph_vlan_interfaces"],
+            "minigraph_port_indices": mgFacts["minigraph_ptf_indices"],
+            "minigraph_portchannels": mgFacts["minigraph_portchannels"],
+            "minigraph_vlans": mgFacts["minigraph_vlans"],
+        })
+
+        logger.info("Copying fdb_info.txt config file to {0}".format(ptfhost.hostname))
+        ptfhost.template(src="fdb/files/fdb.j2", dest=self.FDB_INFO_FILE)
+
+    @pytest.fixture(scope="class", autouse=True)
+    def clearSonicFdbEntries(self, duthosts, rand_one_dut_hostname):
+        """
+            Clears SONiC FDB entries before and after test
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Returns:
+                None
+        """
+        duthost = duthosts[rand_one_dut_hostname]
+        duthost.shell(argv=["sonic-clear", "fdb", "all"])
+
+        yield
+
+        duthost.shell(argv=["sonic-clear", "fdb", "all"])
+
+    @pytest.fixture(scope="class", autouse=True)
+    def validateDummyMacAbsent(self, duthosts, rand_one_dut_hostname):
+        """
+            Validates that test/dummy MAC entry is absent before the test runs
+
+            Args:
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Returns:
+                None
+        """
+        duthost = duthosts[rand_one_dut_hostname]
+        pytest_assert(self.__getFdbTableCount(duthost, self.DUMMY_MAC_PREFIX) == 0, "Test dummy MAC is already present")
+
+    @pytest.fixture(scope="class", autouse=True)
+    def prepareDut(self, request, duthosts, rand_one_dut_hostname):
+        """
+            Prepare DUT for FDB test
+
+            It update the fdb_aging_time value, update the swss configuration, and restore SWSS configuration afer
+            test completes
+
+            Args:
+                request (Fixture): pytest request object
+                duthost (AnsibleHost): Device Under Test (DUT)
+
+            Returns:
+                None
+        """
+        duthost = duthosts[rand_one_dut_hostname]
+        fdbAgingTime = request.config.getoption('--fdb_aging_time')
```

Comment:
@lipxu Is fdb_aging_time a mandatory option for running this case? If nightly test doesn't have this option, will the case use a default value?
Why design it to be request config option? Could it be easy to define it as a variable in case?

Reply:
Correct, this is leverage from another case, we don't need this, will use a fix default value here.